### ISSUE TITLE
`FullScreenSidePanel` focus is inconsistent in Firefox

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9193]
## Observed behavior
In all side panels, there are varying behaviors for focus. Particularly with the `FullScreenSidePanel` component, for example, some of the inconsistencies are:

Exercises/HTML5 exercises
- Folder resources (only the close button, and after main content has been focused through)
- Info panel (no focus)

Browsing a channel
- Info panel (never gets to the close icon)
![2022-03-14 15 24 13](https://user-images.githubusercontent.com/13563002/158270929-5b07900c-6d3f-4329-8682-ca93b55c2772.gif)

## Errors and logs
None

## Expected behavior
Focus behavior should be consistent.

## User-facing consequences
Confusing and user is unable to access all parts of the app via keyboard navigation.

## Steps to reproduce
There are many places where the side panel is open for a learner. Navigate to any of those places (through info icon in a content card, for example).

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
Closed with #9408
